spec/dealership_spec.rb

```diff
@@ -0,0 +1,16 @@
+require './lib/car'
+require './lib/dealership'
+
+RSpec.describe Dealership do
+  it 'exists' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+
+    expect(dealership).to be_a Dealership
+  end
+
+  it 'has attributes' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
```

Comment:
Nice job adding this test for the attributes of a dealership even though it wasn't explicitly stated to do so.